ci-operator/config/devfile/registry/devfile-registry-main__v4.12.yaml

```diff
@@ -21,6 +21,12 @@ releases:
       include_built_images: true
       name: "4.12"
       namespace: ocp
+  multi-latest:
```

Comment:
Since we did nt add release:multi-latest as dep in hypershift-hostedcluster-create-hostedcluster-ref.yaml, this might not be required I believe.